src/test/java/org/zalando/riptide/WithUrlTest.java

```diff
@@ -0,0 +1,145 @@
+package org.zalando.riptide;
+
+/*
+ * ⁣​
+ * Riptide
+ * ⁣⁣
+ * Copyright (C) 2015 Zalando SE
+ * ⁣⁣
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ​⁣
+ */
+import java.util.HashMap;
+import java.util.Map;
+import org.springframework.test.web.client.MockRestServiceServer;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.http.HttpHeaders;
+import org.junit.Test;
+
+import static java.util.Collections.emptyMap;
+import static org.springframework.http.HttpMethod.GET;
+import static org.springframework.test.web.client.match.MockRestRequestMatchers.requestTo;
+import static org.springframework.test.web.client.response.MockRestResponseCreators.withSuccess;
+
+public class WithUrlTest {
+
+    private final RestTemplate template;
+    private final Rest unit;
+    private final MockRestServiceServer server;
+
+    public WithUrlTest() {
+        this.template = new RestTemplate();
+        this.server = MockRestServiceServer.createServer(template);
+        this.unit = Rest.create(template);
+    }
+
+    @Test
+    public void shouldExpandWithoutVariables() {
+        expectRequestTo("https://api.example.com/123");
+
+        unit.withUrl("https://api.example.com/123")
```

Comment:
> I wanted to make it a nonbreaking change for current API

That is still possible, since it would overload all execute methods without a URI as the second parameter.